core/src/test/java/nu/yona/server/analysis/entities/DayActivityTest.java

```diff
@@ -46,6 +55,52 @@ private DayActivity createDayActivity(WeekActivity w, int plusDays)
 				w.getStartTime().plusDays(plusDays).toLocalDate());
 	}
 
+	@Test
+	public void getSpread_beforeTransitionIntoDst_noCorrectionApplied()
+	{
+		DayActivity d = createDayActivity(ZonedDateTime.of(2018, 3, 25, 0, 0, 0, 0, testZone));
+		addActivity(d, "01:05:00", "01:07:00");
+
+		assertSpreadItems(d.getSpread(), "3=0,4=2,5=0");
+	}
+
+	@Test
+	public void getSpread_duringTransitionIntoDst_negativeCorrectionApplied()
+	{
+		// This is a hypothetical situation, as the clock jumps from 2:00 to 3:00
+		DayActivity d = createDayActivity(ZonedDateTime.of(2018, 3, 25, 0, 0, 0, 0, testZone));
+		addActivity(d, "02:05:00", "02:07:00");
+
+		assertSpreadItems(d.getSpread(), "11=0,12=2,13=0");
+	}
+
+	@Test
+	public void getSpread_afterTransitionIntoDst_negativeCorrectionApplied()
+	{
+		DayActivity d = createDayActivity(ZonedDateTime.of(2018, 3, 25, 0, 0, 0, 0, testZone));
+		addActivity(d, "03:05:00", "03:07:00");
+
+		assertSpreadItems(d.getSpread(), "11=0,12=2,13=0");
+	}
+
+	@Test
+	public void getSpread_beforeTransitionOutOfDst_noCorrectionApplied()
```

Comment:
How about adding a test `getSpread_duringTransitionOutOfDst_...`?

Reply:
That cannot occur. Transit out of DST causes an overlap, not a gap.

Reply:
What I meant is during the overlapping period.

Reply:
The users might be doing activities between 2:00 and 3:00 twice, potentially causing overlapping activities. This is not special for us and we have a test already (``getSpreadGetTotalActivityDurationMinutes_multipleActivitiesOverlappingUnsorted_countsOverlappingActivitiesOnce``).